apps/native-component-list/App.tsx

```diff
@@ -2,15 +2,10 @@ import * as SplashScreen from 'expo-splash-screen';
 import * as React from 'react';
 import { Platform, StatusBar } from 'react-native';
 import { AppearanceProvider } from 'react-native-appearance';
-import { enableScreens } from 'react-native-screens';
 
 import RootNavigation from './src/navigation/RootNavigation';
 import loadAssetsAsync from './src/utilities/loadAssetsAsync';
 
-if (Platform.OS === 'android') {
-  enableScreens(true);
```

Comment:
depending on whether we encounter issues here we may want to do:

```
if (Platform.OS === 'ios') {
  enableScreens(false);
}
```

but let's see what happens